feconf.py

```diff
@@ -367,7 +367,7 @@ def get_empty_ratings():
 ENABLE_MAINTENANCE_MODE = False
 
 # Disables all the new structures' pages, till they are completed.
-ENABLE_NEW_STRUCTURES = False
+ENABLE_NEW_STRUCTURES = True
```

Comment:
Can you merge with the most recent version in develop? SInce, this is no longer used.

Reply:
@arod0214 FYI, the instructions for doing this are at step 5 in [this section](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change). Let us know if you need any help!